### PR TITLE
FDA approval fixes #766

### LINKED ISSUE #766: model FDA approval in response to minihackathon
Script Query: “Find FDA Approved EGFR inhibitors”, aka: “FDA Approved Chemical Entities that ‘negatively regulate’ EGFR” 
Four options:
- As a node property (ie: Chemical Entities would have an optional boolean slot called ‘FDA approval’)
- As an ‘edge property’ that qualifies the ‘treats’, ‘approved to treat’ or ‘entity negatively regulates entity’ predicate, etc.
- As a predicate ‘FDA approved to treat’ (child of ‘approved to treat’). 
‘approved to treat’ already has guidance on ‘not approved to treat’ modeling.
“FDA approved ChemicalEntity that treats EGFR expression” - strange. :) 
- As a new node (child of abstract class ‘Information Content Entity’?) 
‘Information Content Entity’ has a lot of slots that might not apply to a node with name(?) ‘FDA Approval’?

### PR DESCRIPTION
**Pull request by sierra-moxon**

### CODE REVIEW

**Comment by linikujp** [comment]
Can you explain what does it mean by "highest" FDA approval status?

Thanks,
Asiyah

**Comment by sierra-moxon** [comment]
Hi @linikujp - great question!  I think in general, modeling 'FDA approval' as an edge property makes the most sense.  However, as a summary property, the "highest" FDA approval status (the closest-to-approved status) for any use (for any disease/procedure) is what we'd store on the ChemicalEntity node itself (as a node property).

**Comment by sierra-moxon** [comment]
Discussed and confirmed direction with Chris B on the helpdesk call.

File: biolink-model.yaml [hunk 1]
```diff
@@ -202,7 +202,105 @@ enums:
         description: nucleic acid
       "AA":
         description: amino acid
-        
+
+  chemical_available_from_enum:
```
Comment (vdancik) [inline biolink-model.yaml:206]: maybe "drug availability" is more precise then "chemical available from"

Comment (sierra-moxon) [reply biolink-model.yaml]: done! :) thanks a bunch.

File: biolink-model.yaml [hunk 2]
```diff
@@ -202,7 +202,105 @@ enums:
...
+        description: >-
+          chemical entity is available over the counter without a prescription.
+      "prescription":
+        description:
+          chemical entity is available by prescription.
+
+  FDA_approval_status_enum:
+    description: >-
+    permissible_values:
+      "FDA Approval Phase 1":
```
Comment (vdancik) [inline biolink-model.yaml:219]: "FDA Approval Phase 1" is easily confused with "FDA Clinical Research Phase 1", I suggest to call this phase "Discovery & Development Phase"

File: biolink-model.yaml [hunk 3]
```diff
@@ -202,7 +202,105 @@ enums:
...
+        description: >-
+          Discovery & Development Phase. Discovery involves researchers finding new possibilities
+          for medication through testing molecular compounds, noting unexpected effects from existing treatments,
+          or the creation of new technology that allows novel ways of targeting medical products to sites in the body.
+          Drug development occurs after researchers identify potential compounds for experiments.
+      "FDA Approval Phase 2":
```
Comment (vdancik) [inline biolink-model.yaml:225]: "FDA Approval Phase 2" is easily confused with "FDA Clinical Research Phase 2", I suggest to call this phase "Preclinical Research Phase".

File: biolink-model.yaml [hunk 4]
```diff
@@ -202,7 +202,105 @@ enums:
...
+        description: >-
+          Preclinical Research Phase.  Once researchers have examined the possibilities a new drug may contain,
+          they must do preliminary research to determine its potential for harm (toxicity).
+          This is categorized as preclinical research and can be one of two types: in vitro or in vivo.
+      "FDA Approval Phase 3":
```
Comment (vdancik) [inline biolink-model.yaml:230]: "FDA Approval Phase 3" is easily confused with "FDA Clinical Research Phase 3", I suggest to call this phase "FDA Clinical Research Phase".

File: biolink-model.yaml [hunk 5]
```diff
@@ -202,7 +202,105 @@ enums:
...
+      "FDA Post-Market Safety Monitoring":
+        description: >-
+          FDA Post-Market Safety Monitoring.  The last phase of drug approval is an ongoing one
+          while the drug is on the marketplace. If a developer wants to change anything about the
+          drug formulation or approve it for a new use, they must apply with the FDA. The FDA also
+          frequently reviews the drug’s advertising and its manufacturing facility to make sure
+          everything involved in its creation and marketing is in compliance with regulations.
+      "FDA Clinical Research Phase 1":
+        description: >-
+          In FDA Approval Phase 3, the Clinical Research Phase 1 involves 20 – 100 study participants and
```
Comment (vdancik) [inline biolink-model.yaml:248]: I suggest to drop  "In FDA Approval Phase 3" and just write "The Clinical Research Phase 1 ..."

File: biolink-model.yaml [hunk 6]
```diff
@@ -202,7 +202,105 @@ enums:
...
+          lasts several months. This phase is used to determine the safety and dosage of the drug,
+          and about 70% of these drugs move on to the next clinical research phase.
+      "FDA Clinical Research Phase 2":
+        description: >-
+          In FDA Approval Phase 3, the Clinical Research Phase 2 involves up to several hundred people,
```
Comment (vdancik) [inline biolink-model.yaml:253]: I suggest to drop  "In FDA Approval Phase 3" and just write "The Clinical Research Phase 2 ..."

File: biolink-model.yaml [hunk 7]
```diff
@@ -202,7 +202,105 @@ enums:
...
+          who must have the disease or condition the drug supposes to treat. This phase can last
+          from a few months to two years, and its purpose is to monitor the efficacy of the drug,
+          as well as note side effects that may occur.
+      "FDA Clinical Research Phase 3":
+        description: >-
+          In FDA Approval Phase 3, the Clinical Research Phase 3 involves 300 – 3000 volunteers
```
Comment (vdancik) [inline biolink-model.yaml:259]: I suggest to drop  "In FDA Approval Phase 3" and just write "The Clinical Research Phase 3 ..."

**Review by vdancik — CHANGES_REQUESTED** [review]

**Comment by vdancik** [comment]
We are missing levels for regular approval and post-approval withdrawal.

File: biolink-model.yaml (hunk 7 above)
Comment (sierra-moxon) [reply biolink-model.yaml:259]: done

File: biolink-model.yaml (hunk 6 above)
Comment (sierra-moxon) [reply biolink-model.yaml:253]: done

File: biolink-model.yaml (hunk 5 above)
Comment (sierra-moxon) [reply biolink-model.yaml:248]: done

File: biolink-model.yaml (hunk 4 above)
Comment (sierra-moxon) [reply biolink-model.yaml:230]: done

File: biolink-model.yaml (hunk 3 above)
Comment (sierra-moxon) [reply biolink-model.yaml:225]: done

File: biolink-model.yaml (hunk 2 above)
Comment (sierra-moxon) [reply biolink-model.yaml:219]: done

File: biolink-model.yaml [hunk 8]
```diff
@@ -1096,6 +1194,33 @@ slots:
     # if this also represents an interval of time.
     range: time type
 
+  is supplement:
+    description: >-
+    is_a: node property
+    range: chemical mixture
+
+  trade name:
```
[the remark this answers is not in the document]
Comment (sierra-moxon) [reply biolink-model.yaml:1202]: @vdancik - trade name seems like a very specific name to me as well.  My impression from our meeting about this was that this was intentionally specific.  We can certainly refactor in the next iteration.

**Pull request closed without merging**